src/server_manager/electron_app/index.ts

```diff
@@ -23,6 +23,9 @@ import type {HttpRequest, HttpResponse} from '../infrastructure/path_api';
 import {fetchWithPin} from './fetch';
 import * as menu from './menu';
 
+// Injected by esbuild/webpack during build
+declare const SENTRY_DSN_: string | undefined;
```

Comment:
I believe in Cordova we read the SENTRY_DSN from a json file.
We should pick one approach and use the same for Electron and Cordova to minimize divergence. It's confusing to see the different approaches.

Reply:
This is manager, we only have electron. In client this is not an issue cause it's already implemented in Webpack.